src/cmd.js

```diff
@@ -1,117 +1,197 @@
 #!/usr/bin/env node
 
-import minimist from 'minimist'
+
+import fs from 'fs'
```

Comment:
Yes, but once you have a ton of dependencies (which is really easy nowadays), this can add up quite substantially.